.github/config.env

```diff
@@ -1,4 +1,4 @@
 JAVA_VERSION=17
 JAVA_DISTRIBUTION=temurin
 MAVEN_VERSION=3.9.9
-HAZELCAST_VERSION=5.3.0
\ No newline at end of file
+HAZELCAST_VERSION=5.4.0
```

Comment:
why not 5.5.0?

Reply:
`5.5.0` broke, which I've [raised seperately](https://github.com/hazelcast/hazelcast-cpp-client/issues/1235) - this is a stepping stone.